admin/replication/ConvertReplicationPacketToXML

```diff
@@ -57,43 +57,43 @@ while (my $p_line = shift_p())
 
 
     my $p_row = decode_row($p_line, qw( SeqId TableName Op XID ));
-    my $op = $p_row->{"Op"} or die "No Op in $p_line";
+    my $op = $p_row->{'Op'} or die "No Op in $p_line";
```

Comment:
There's no need to use quotes here either way FWIW.  Same for the other cases below.